views/default/input/button.php

```diff
@@ -1,34 +1,41 @@
 <?php
+
 /**
- * Create a input button
- *
- * @package Elgg
- * @subpackage Core
+ * Renders a <button>
  *
- * @uses $vars['src']   Src of an image
```

Comment:
The view didn't resolve the type from src attribute, so it would have to been both type and src, in which case the attributes will resolve as they should.